Server/VikingVault.API/Controllers/PaymentTogglingController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VikingVault.Services;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.API.Controllers
+{
+   
+    [Route("[controller]")]
+    [ApiController]
+    public class PaymentTogglingController : ControllerBase
+    {
+        private readonly IPaymentTogglingService _paymentTogglingService;
+
+        public PaymentTogglingController(IPaymentTogglingService paymentTogglingService)
+        {
+            _paymentTogglingService = paymentTogglingService;
+        }
+
+        [HttpGet("{id}", Name = "Get")]
+        public ActionResult<Boolean> Get(int id)
+        {
+            bool? isPaymentEnabled = _paymentTogglingService.IsPaymentEnabled(id);
+            if (isPaymentEnabled == null)
```

Comment:
Even for a single line in the if statement, it is a good practice to use braces {}. 
Here you can even use the conditional operator: return isPaymentEnabled == null ? StatusCode(500,"Internal Server Error") : Ok(isPaymentEnabled)